src/main/java/Duke.java

```diff
@@ -1,10 +1,71 @@
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+    public static final int TODO_DESCRIPTION_START_INDEX = 5;
+    public static final int DEADLINE_DESCRIPTION_START_INDEX = 9;
+    public static final int DEADLINE_BY_SPACE_LENGTH = 4;
+    public static final int EVENT_DESCRIPTION_START_INDEX = 6;
+    public static final int EVENT_FROM_SPACE_LENGTH = 6;
+    public static final int EVENT_TO_SPACE_LENGTH = 4;
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        System.out.println("Hello! I'm Gary");
+        System.out.println("What can I do for you?");
+
+        Task[] todos = new Task[100];
+        int todosCount = 0;
+
+        String line;
+        line = in.nextLine();
+        String[] lineWords;
```

Comment:
The variable name of lineWords is a bit confusing as it can be a variety of string within the code, I believe it's better to store each member of lineWords into a variable that is easier to understand. e.g.:
`String lineToMark = lineWords[0]` then for subsequent code, use `lineToMark`.